prime-dotnet-webapi/Controllers/EnrolleePaperSubmissionsController.cs

```diff
@@ -263,9 +263,9 @@ public async Task<ActionResult> SetEnrolleePaperSubmissionProfileCompleted(int e
         [ProducesResponseType(typeof(ApiResultResponse<EnrolleeAdjudicationDocument>), StatusCodes.Status200OK)]
         public async Task<ActionResult> GetAdjudicationDocuments(int enrolleeId)
         {
-            if (!await _enrolleePaperSubmissionService.PaperSubmissionIsEditableAsync(enrolleeId))
+            if (!await _enrolleeService.EnrolleeExistsAsync(enrolleeId))
```

Comment:
Was this intentional? Should this be checking if it is a paper enrollee not just an enrollee in general